okhttp/src/test/java/okhttp3/CallTest.java

```diff
@@ -112,7 +113,7 @@
   private RecordingEventListener listener = new RecordingEventListener();
   private HandshakeCertificates handshakeCertificates = localhost();
   private OkHttpClient client = clientTestRule.newClientBuilder()
-      .eventListener(listener)
+      .eventListenerFactory(clientTestRule.wrap(listener))
```

Comment:
This could be simplified by putting the eventListener call on ClientTestRule, before getting the builder or client.